API/Validators/CustomerValidator.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Data.Entities;
+using FluentValidation;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Data.Validators
+{
+    internal class CustomerValidator : AbstractValidator<Customer>
+    {
+        public CustomerValidator()
+        {
+            RuleFor(customer => customer.FullName)
+                .NotNull()
+                .MinimumLength(5);
+
+            RuleFor(customer => customer.Email)
+                .NotEmpty()
+                .Equal(costumer => costumer.EmailConfirmation)
+                .WithMessage("Emails don't match")
+                .Matches(@"^([\w\.\-]+)@([\w\-]+)((\.(\w){2,3})+)$")
```

Comment:
```suggestion
            .EmailAddress(EmailValidationMode.Net4xRegex);
```

Remover regex e utilizar o próprio fluent validation